application/frontend/src/pages/releases/detail/information-box.tsx

```diff
@@ -33,54 +33,77 @@ import { useLingui } from "@lingui/react";
  * @constructor
  */
 export const InformationBox: React.FC<Props> = ({ releaseData, releaseId }) => {
-  const alertBoxClasses = "border-4 rounded-2xl p-4 text-center mb-2";
+  // a right aligned list of all our datasets and their visualisation colour/box
+  const DatasetList = () => (
+    <ul className="text-right">
+      {Array.from(releaseData.datasetMap.entries()).map(([uri, vis], index) => (
+        <li key={index} className="flex flex-row justify-end align-middle">
+          <span className="mr-6 font-mono">{uri}</span>
+          <span className="h-6 w-6">{vis}</span>
+        </li>
+      ))}
+    </ul>
+  );
+
+  const ActivateDeactivateButtonRow = () => (
+    <div className="flex flex-row space-x-4">
+      <button
+        className="btn-success btn-lg btn"
+        disabled={!!releaseData.activation}
+      >
+        Activate Release
+      </button>
+      <button
+        className="btn-warning btn-lg btn"
+        disabled={!!!releaseData.activation}
+      >
+        Deactivate Release
+      </button>
+    </div>
+  );
 
   return (
-    <Box heading="Release Information">
+    <Box heading={`${releaseData.applicationDacTitle}`}>
```

Comment:
Assuming `releaseData.applicationDacTitle` is a string:
```suggestion
    <Box heading={releaseData.applicationDacTitle}>
```

Reply:
Yep - that one had some extras in the heading string that I must have removed